examples/auto_ml_checkpoint.py

```diff
@@ -97,14 +99,14 @@ def main(tmpdir, sleep_time: float = 0.001, n_iter: int = 10):
         ('sleep_2', ForEachDataInput(Sleep(sleep_time))),
         ('checkpoint2', ExpandDim(DefaultCheckpoint())),
         ('multiplication_3', MultiplyByN())
-    ], cache_folder=tmpdir).set_hyperparams_space(HYPERPARAMETER_SPACE)
+    ], cache_folder=resumable_pipeline_folder).set_hyperparams_space(HYPERPARAMETER_SPACE)
 
     time_a = time.time()
     auto_ml = AutoML(
         pipeline,
-        refit_trial=False,
+        refit_trial=True,
```

Comment:
I've already made the comment in the bast, but it'd be good to provide some example comment for the time printed so that readers can realize how faster it is. Optional fix.